sign/sign.go

```diff
@@ -299,6 +394,40 @@ func (s *Signer) IsImageSigned(imageRef string) (bool, error) {
 	return len(signatures) > 0, nil
 }
 
+// IsFileSigned takes an path reference and retrusn true if there is a signature
+// available for it. It makes no signature verification, only checks to see if
+// there is a TLog to be found on Rekor.
+func (s *Signer) IsFileSigned(ctx context.Context, ko cliOpts.KeyOpts, path string) (bool, error) { // nolint: gocritic
+	rClient, err := s.impl.NewRekorClient(ko.RekorURL)
+	if err != nil {
+		return false, fmt.Errorf("creating rekor client: %w", err)
+	}
+
+	blobBytes, err := s.impl.PayloadBytes(path)
+	if err != nil {
+		return false, err
+	}
+
+	uuids, err := s.impl.FindTLogEntriesByPayload(ctx, rClient, blobBytes)
+	if err != nil {
+		return false, fmt.Errorf("find rekor tlog entries: %w", err)
+	}
+
+	return len(uuids) > 0, nil
+}
+
+// FileSha256 return a sha256 has of a file based on the
+func (s *Signer) FileSha256(path string) (string, error) {
```

Comment:
We already have this function in the release-utils: https://github.com/kubernetes-sigs/release-utils/blob/11d18a85425eeb3024f2f82030a8382fdff74277/hash/hash.go#L38-L41

Reply:
Great I'll replace it.